qemu/tests/win_heavyload.py

```diff
@@ -129,7 +136,7 @@ def add_option(cmd, key, val):
 
         error_context.context("Verify vm is alive", logging.info)
         utils_misc.wait_for(vm.verify_alive,
-                            timeout=test_timeout, step=steping)
+                            timeout=test_timeout + 2, step=steping)
```

Comment:
Why do we need add 2 seconds for timeout ? I'm not sure if it is ok to put an absolute value here.

Reply:
"test_timeout"  is the "Duration" to run heavyload test too. But it need few second to star the test, and then run the test with for $Duration min.  If wait_for()  only "test_timeout",  the heavyload test may still not finished,  mandatory exist the test by force will cause the huge size temp file can not be deleted.   
I have test many times,  2 second is long enough.  If the process still can not exist successfully, it will be killed by force in below.
And in this patch, it changed to save temp file to a new data disk instead of system disk.  Even if quit the test by force and temp file can not be delete automatically,  it will not affect the whole system, and the other case after it.